backend/api/Areas/Admin/Controllers/AccessRequestController.cs

```diff
@@ -24,23 +23,19 @@ namespace Pims.Api.Areas.Admin.Controllers
     public class AccessRequestController : Controller
     {
         #region Properties
-        private readonly IPimsAdminService _pimsAdminService;
+        private readonly IPimsService _pimsService;
         private readonly IMapper _mapper;
-        private readonly IUserService _userService;
         #endregion
 
-        #region Construstor
+        #region Constructors
         /// <summary>
         /// Creates a new instance of an AccessRequestController object, initializes with specified parameters.
         /// </summary>
-        /// <param name="pimsAdminService"></param>
+        /// <param name="pimsService"></param>
         /// <param name="mapper"></param>
-        /// <param name="userService"></param>
-        public AccessRequestController(IPimsAdminService pimsAdminService,
-            IMapper mapper, IUserService userService)
+        public AccessRequestController(IPimsService pimsService, IMapper mapper)
```

Comment:
What is the background on removing the pimsAdminService here?

Reply:
is everything a child of pimsService now?

Reply:
There is no `pimsAdminService` anymore.  Anything left after this refactor has been moved into `pimsService`.